x/auth/vesting/types/vesting_account.go

```
@@ -518,3 +584,8 @@ func (plva PermanentLockedAccount) Validate() error {
 
 	return plva.BaseVestingAccount.Validate()
 }
+
+// UpdateSchedule updates the vesting schedule for a permanent locked account.
+func (plva *PermanentLockedAccount) UpdateSchedule(blockTime time.Time, amount sdk.Coins) error {
+	return nil
+}
```

[blocking] this should probably return an error.

---

an error could cause a issue upstream as this should not error. If we error here then upstream needs to handle unimplemented updates schedules which will be treated the same as nil.

---

hmm okay then should the comment say this is a no-op?